vue-frontend/src/pages/portfolio/[id].vue

```diff
@@ -0,0 +1,23 @@
+<template>
+  <div>
+    <Header />
+    <LandingSection />
+  </div>
+</template>
+
+<script>
+import Header from "@/components/partials/ScreenHeaderV2.vue";
+import LandingSection from "@/components/portfolio/LandingSection.vue";
+
+export default {
+  data() {
+    return {
+      id: this.$route.params.id,
+    };
+  },
+  components: {
+    Header,
+    LandingSection,
```

Comment:
I think the Landing section is part of another ticket, We have to add logic that how we load json data based on the parameter of the URL.

Reply:
Add that section only reference purpose